plugins/source/gcp/resources/services/networkconnectivity/internal_ranges.go

```diff
@@ -0,0 +1,49 @@
+package networkconnectivity
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v14/arrow"
+	"google.golang.org/api/networkconnectivity/v1"
+
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func InternalRanges() *schema.Table {
```

Comment:
We saw our share of bugs related to listing relation tables twice (as relations and in the top-level), so it's a good idea to make relation tables unexported.
```suggestion
func internalRanges() *schema.Table {
```

Reply:
done